<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principale</class>
 <widget class="QWidget" name="principale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>306</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="ca_bours">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="vol_bours">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_bours">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>560</y>
     <width>306</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="vol_assur">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_assur">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_assur">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>306</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="vol_banq">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_banq">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_banq">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Agence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="agence">
        <item>
         <property name="text">
          <string>Toutes Agences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mulhouse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marseille</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paris</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Strasbourg</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Banquier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="banquier">
        <item>
         <property name="text">
          <string>Tous Banquiers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ahmed Benmoussa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alexandre Maurice</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alice Benarfa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yanis Moujab</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,3">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="pdp">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/assets/pic.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>ALEX MOOR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,3,1">
    <item>
     <widget class="QLabel" name="nom_agence">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>BNR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="image">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/assets/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nom_banquier">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="elargir_bq">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="elargir_assur">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="elargir_bours">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>270</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_bours">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_assur">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_bq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>produit_bours</zorder>
  <zorder>produit_assur</zorder>
  <zorder>produit_bq</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>elargir_bq</zorder>
  <zorder>elargir_assur</zorder>
  <zorder>elargir_bours</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qtcharts/qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
